src/abi/tests.rs

```diff
@@ -649,15 +726,26 @@ fn constructor() {
     assert!(idl.instructions[0].docs.is_none());
     assert_eq!(
         idl.instructions[0].accounts,
-        vec![IdlAccountItem::IdlAccount(IdlAccount {
-            name: "data_account".to_string(),
-            is_mut: true,
-            is_signer: false,
-            is_optional: Some(false),
-            docs: None,
-            pda: None,
-            relations: vec![],
-        })]
+        vec![
+            IdlAccountItem::IdlAccount(IdlAccount {
+                name: "data_account".to_string(),
+                is_mut: true,
+                is_signer: false,
+                is_optional: Some(false),
+                docs: None,
+                pda: None,
+                relations: vec![],
+            }),
+            IdlAccountItem::IdlAccount(IdlAccount {
+                name: "systemProgram".to_string(),
```

Comment:
I wrote `data_account` and you wrote `systemProgram`. We should adopt either camelCase or snake_case for naming the accounts, but not both.

Reply:
Good point. Renamed to `dataAccount`